lib/qubinode_menu_options.sh

```diff
@@ -21,6 +21,7 @@ function qubinode_product_deployment () {
               fi
               ;;
           ocp4)
+              openshift4_prechecks
```

Comment:
Did you add this because you got errors about not finding the vars file like ocp4.yml? If yes, i fix this in the openshift4_enterprise_deployment, the prechecks should be called from openshift4_enterprise_deployment.